docs/standard/microservices-architecture/multi-container-microservice-net-applications/rabbitmq-event-bus-development-test-environment.md

```diff
@@ -30,7 +30,7 @@ The RabbitMQ implementation of a sample dev/test event bus is boilerplate code.
 
```

Comment:
do you mind closing the class definition from line 23? Otherwise, GitHub doesn't format the rest of the text properly